src/t_stream.c

```diff
@@ -3356,7 +3357,8 @@ void xautoclaimCommand(client *c) {
         int moreargs = (c->argc-1) - j; /* Number of additional arguments. */
         char *opt = c->argv[j]->ptr;
         if (!strcasecmp(opt,"COUNT") && moreargs) {
-            if (getRangeLongFromObjectOrReply(c,c->argv[j+1],1,LONG_MAX,&count,"COUNT must be > 0") != C_OK)
+            long max_count = LONG_MAX / (max(sizeof(streamID), attempts_factor));
```

Comment:
@oranagra Why we need to use max?
streamID is always 16 and attempts_factor is always 10. I think it is never changed.

Reply:
@oranagra I think it is better to extract max_count outside of while loop.

Reply:
i was paranoid about attempts_factor being changed some day, and then this assumption you suggest breaks and no one notices.
in practice this max will be resolved at compile time, we don't need to worry about moving it out of the loop